Vagrantfile

```diff
@@ -79,6 +80,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
             mesos_local_address: node[:ip],
             marathon_local_address: node[:ip],
             consul_join: consul_join,
+            consul_retry_join: [consul_retry_join],
+            consul_bootstrap_expect: 1,
```

Comment:
if it should be 3, no need to specify it as it will be 3 servers in the group (above in ansible groups)